components/inputs/test/input-date-time-range.visual-diff.js

```diff
@@ -162,6 +167,7 @@ describe('d2l-input-date-time-range', () => {
 		async function changeInnerInputDateTime(page, selector, inputSelector, date, waitForTime) {
 			return page.$eval(selector, (elem, inputSelector, date, waitForTime) => {
 				const dateElem = elem.shadowRoot.querySelector(inputSelector);
+				if (dateElem.value.substring(0, 23) === date.substring(0, 23)) return; // Needed for retries
```

Comment:
If the date is already what we want, just cancel out.  I had to strip off the `Z` that was sometimes there, sometimes wasn't